### DISCUSSION TITLE
Installing Packer.nvim via chocolatey

### DISCUSSION THREAD

**Discussion by Nowaaru** [wbthomason/packer.nvim#878] [Q&A] [answered]
spent three hours trying to install this and I can't figure out how. can someone walk me through installing packer.nvim, or should I just use vim-plug?

**Comment by bryant-the-coder** [comment]
git clone... its as easy as cloning a repo... did you read the  docs / README?

**Comment by bryant-the-coder** [comment]
`git clone https://github.com/wbthomason/packer.nvim "$env:LOCALAPPDATA\nvim-data\site\pack\packer\start\packer.nvim"`

if on windows

**Reply by Nowaaru** [reply]
> `git clone https://github.com/wbthomason/packer.nvim "$env:LOCALAPPDATA\nvim-data\site\pack\packer\start\packer.nvim"`
> 
> if on windows

i did read the docs. i've also tried putting it in `C:\tools\neovim\nvim-win64\share\nvim\runtime\lua`, `C:\tools\neovim\nvim-win64\share\nvim\runtime\pack\dist\opt`. none give the availability of Packer's commands.

**Comment by Nowaaru** [comment, marked as answer]
ah, got it! apparently i had to uninstall nvim, clear all leftover files, reinstall, then install packer, and then restart my computer. why? don't know. problem's fixed, though! 👍
